### PR TITLE
TYPO3 10 compatibility

### PR DESCRIPTION
**Pull request by kitzberger**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,9 +20,9 @@
     }
   ],
   "require": {
-    "typo3/cms-core": "^8.7.13",
-    "typo3/cms-indexed-search": "^8.7.13",
-    "typo3/cms-scheduler": "^8.7.13"
+    "typo3/cms-core": "^8.7 || ^9.5 || ^10.4",
```
Comment (WebsiteDeveloper) [inline composer.json]: With the Proposed Changes 8.7 would no longer be supported.
I think you can remove it though i will add secondary branch for the legacy version

File: composer.json [hunk 2]
```diff
@@ -31,5 +31,10 @@
     "psr-4": {
       "KKSoftware\\IndexedSearchGC\\": "Classes"
     }
+  },
+  "extra": {
+    "typo3/cms": {
+      "extension-key": "indexex_search_gc"
```
Comment (WebsiteDeveloper) [inline composer.json]: There seems to be a typo here

File: ext_emconf.php [hunk 3]
```diff
@@ -14,9 +14,9 @@
     'version' => '1.0.0',
     'constraints' => [
         'depends' => [
-            'typo3' => '8.7.0-8.7.99',
-            'indexed_search' => '8.7.0-8.7.99',
-            'scheduler' => '8.7.0-8.7.99'
+            'typo3' => '8.7.0-10.4.99',
```
Comment (WebsiteDeveloper) [inline ext_emconf.php]: Same as in the composer file. Just support typo3 9.5 as minimal version

**Review by WebsiteDeveloper — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (kitzberger) [reply composer.json]: @WebsiteDeveloper, oops ;-) Fixed now!

File: composer.json (hunk 1 above)
Comment (kitzberger) [reply composer.json]: @WebsiteDeveloper, fixed.

**Review by WebsiteDeveloper — APPROVED** [review]

**Comment by WebsiteDeveloper** [comment]
@kitzberger Thanks for the contribution
I will release a new version on packagist and TER later today

**Pull request merged**